guides/common/modules/con_additional-maintenance-required-for-load-balancing.adoc

```diff
@@ -0,0 +1,10 @@
+[id="additional-maintenance-required-for-load-balancing_{context}"]
+= Additional maintenance required for load balancing
+
+Configuring {SmartProxies} to use a load balancer results in a more complex environment and requires additional maintenance.
+
+The following additional steps are required for load balancing:
+
+* You must ensure that all {SmartProxies} have the same content views and synchronize all {SmartProxies} to the same content view versions.
+* You must upgrade each {SmartProxy} in sequence.
+* You must back up each {SmartProxy} that you configure regularly.
```

Comment:
what does "regularly" mean? If I do not use LB, can I skip backups?

Maybe something like "backup all SP behind LB even if you think their identical" in different words?

Reply:
This looks like content that has been here for years so based on the comment above, let's try removing it.
```suggestion
```